ow_plexil/config/fault_dependencies.yaml

```diff
@@ -0,0 +1 @@
+fault_dependencies_verbose: false
```

Comment:
I suggest naming this file something generic like `system.yaml` or `package.yaml` so that it can be good for any configuration options...unless it is a ROS convention to keep related configurations in a specifically named file.

In any case, a comment explaining the option would be useful.